src/content/recipes/tailwindcss.md

```diff
@@ -0,0 +1,188 @@
+<div class="aside aside__no-top">
+
+This recipe assumes that you have a React app using Tailwindcss and have just set up Storybook >=6.0 using the [getting started guide](/docs/react/get-started/install). Don’t have this? Follow Tailwind's [setup instructions](https://tailwindcss.com/docs/installation) then run:
+
+```shell
+# Add Storybook:
+npx sb init
+```
+
+</div>
+
+<RecipeHeader>
+
+How to setup Tailwind CSS and Storybook
+
+</RecipeHeader>
+
+Storybook.js is a fantastic tool for developing and showcasing UI components in isolation. One of the great things about it is that you can use any CSS framework you like, including tailwind CSS.
+
+In this post, we'll go over:
+
+1. 🏗️ Build Tailwind next to Storybook
+2. 🎁 Provide Tailwind to stories
+3. 🧱 Use Tailwind in your components
+4. 🎨 Switch Tailwind themes in a click
+
+![Finished example of Tailwind CSS in Storybook with a theme switcher](https://user-images.githubusercontent.com/18172605/208201389-1f448dbb-978c-442e-9d6b-7bf3fea63e64.gif)
+
+## Build Tailwind next to Storybook
+
+To develop your tailwind alongside your stories, you’ll need a development environment that runs two independent but coordinated processes using `concurrently`
+
+```shell
+# install concurrently:
+yarn add --dev concurrently
+```
+
+Then add these updated scripts to your `package.json`
+
+```json
+"scripts": {
+   "//": "New scripts to run and build Storybook with Tailwind",
+   "storybook": "concurrently \"yarn:watch:*\"",
+   "build-storybook": "concurrently \"yarn:build:*\"",
+   "build:css": "npx tailwindcss -i ./src/tailwind.css -o ./public/tailwind.css",
+   "build:storybook": "storybook build",
+   "watch:css": "npx tailwindcss -i ./src/tailwind.css -o ./public/tailwind.css --watch",
+   "watch:storybook": "storybook dev -p 6006"
+ },
+```
+
+## Import the tailwind.css file into Storybook
+
+Now you can import the tailwind.css file into your .storybook/preview-head.html file. This will make Tailwind’s style classes available to all of your stories.
```

Comment:
```suggestion
Now you can import the tailwind.css file into your `.storybook/preview-head.html` file. This will make Tailwind’s style classes available to all of your stories.
```